scripts/reports/repo-health.sh

```diff
@@ -22,6 +22,8 @@ set -eo pipefail
 
 if [ -z "$GH_OATH" ]; then echo "GH_OATH is not set. Existing..."; exit 1; fi
 if [ -z "$CODACY_OATH" ]; then echo "CODACY_OATH is not set. Existing..."; exit 1; fi
+if [ -z "$CORTX_REPO" ]; then echo "CODACY_OATH is not set. Existing..."; exit 1; fi
```

Comment:
change `CODACY_OATH` to var in error message.

Reply:
Fixed. Please review